libs/designer/src/lib/ui/templates/templatesStrings.ts

```diff
@@ -38,3 +39,55 @@ export const useTemplatesStrings = () => {
     },
   };
 };
+
+export const useConnectorStatusStrings = () => {
+  const intl = useIntl();
+
+  return {
+    connected: intl.formatMessage({
+      defaultMessage: 'Connected',
+      id: 'oOGTSo',
+      description: 'Connected text',
+    }),
+    notConnected: intl.formatMessage({
+      defaultMessage: 'Not connected',
+      id: 'WxJJcQ',
+      description: 'Not Connected text',
+    }),
+    authenticated: intl.formatMessage({
+      defaultMessage: 'Authenticated',
+      id: 'iwKxSD',
+      description: 'Connection authenticated text',
+    }),
+    notAuthenticated: intl.formatMessage({
+      defaultMessage: 'Not authenticated',
+      id: 'QdRn5z',
+      description: 'Connection not authenticated text',
+    }),
+  };
+};
+
+export const getDetailTriggerDisplayValue = (triggers: LogicAppsV2.Triggers): string => {
+  const intl = getIntl();
+  const detailTriggerStrings = {
+    Request: intl.formatMessage({
+      defaultMessage: 'Request',
+      id: 'VOk0Eh',
+      description: 'Trigger belongs to Request category',
+    }),
+    Recurrence: intl.formatMessage({
+      defaultMessage: 'Recurrence',
+      id: 'CdyJ6f',
+      description: 'Trigger belongs to Recurrence category',
+    }),
+    Event: intl.formatMessage({
+      defaultMessage: 'Event',
+      id: '2iq12E',
+      description: 'Trigger belongs to Event category',
+    }),
+  };
+
+  return Object.values(triggers)
+    .map((trigger) => detailTriggerStrings[trigger.type as keyof typeof detailTriggerStrings] ?? detailTriggerStrings.Event)
+    .join(', ');
+};
```

Comment:
Apart from resource strings i already created a method to getTriggerType in my last PR, can you use that and then check with Divya on what we need to call all types.